land_use/views.py

```diff
@@ -0,0 +1,139 @@
+    """Returns the list of `models.LandUseClasses` spatial data.
+
+    Filters:
+        * co_cr (list): filtering Regional Coordenation using code.
+        * co_funai (list): filtering Indigenou Lands using Funai code.
+        * year_map (list): filtering years mapped in land use mapping.
+        * in_bbox (bbox): bounding box
+            (min lon, min lat, max lon, max lat).
+    """
+    queryset = models.LandUseClasses.objects.all()
+    serializer_class = serializers.LandUseSerializer
+    bbox_filter_field = 'geom'
+    filterset_class = land_use_filters.LandUseClassesFilter
+    filter_backends = (
+        rest_framework.DjangoFilterBackend,
+        gis_filters.InBBoxFilter,
+    )
+
+
+class LandUseDetailView(generics.RetrieveAPIView):
+    """Returns detailed data for a queried element of `models.LandUseClasses` data.
+
+    Filters:
+        * id (int): filtering request poligon identifier.
+    """
+    queryset = models.LandUseClasses.objects.all()
+    serializer_class = serializers.LandUseDetailSerializer
+    lookup_field = 'id'
+    filter_backends = (rest_framework.DjangoFilterBackend,)
+
+
+class LandUseYearsView(generics.ListAPIView):
+    """Return list of years that have land use mapping for filters applied of 
+    `models.LandUseClasses` data."""
+    queryset = models.LandUseClasses.objects.distinct('nu_ano')
+    serializer_class = serializers.LandUseYearsSerializer
+    bbox_filter_field = 'geom'
+    filterset_class = land_use_filters.LandUseClassesFilter
+    filter_backends = (rest_framework.DjangoFilterBackend,)
+
+
+class LandUseTableView(generics.ListAPIView):
+    """Returns list data without geometry from 'models.LandUseClasses' data.
+
+    Filters:
+        * co_cr (list): filtering Regional Coordenation using code.
+        * co_funai (list): filtering Indigenou Lands using Funai code.
+        * year_map (list): filtering years mapped in land use mapping.
+        * in_bbox (bbox): bounding box
+            (min lon, min lat, max lon, max lat).
+    """
+    queryset = models.LandUseClasses.objects.all()
+    serializer_class = serializers.LandUseTableSerializer
+    bbox_filter_field = 'geom'
+    filterset_class = land_use_filters.LandUseClassesFilter
+    filter_backends = (rest_framework.DjangoFilterBackend,)
+
+
+class LandUseClassesView(generics.ListAPIView):
+    """Flag list classification stages adopted in the mapping of land use of 
+    `models.LandUseClasses` existing in the applied filters.
+
+    Filters:
+        * co_cr (list): filtering Regional Coordenation using code.
+        * co_funai (list): filtering Indigenou Lands using Funai code.
+        * year_map (list): filtering years mapped in land use mapping.
+        * in_bbox (bbox): bounding box
+            (min lon, min lat, max lon, max lat).
+    """
+    queryset = models.LandUseClasses.objects.distinct('no_estagio')
+    serializer_class = serializers.LandUseClassesSerializer
+    bbox_filter_field = 'geom'
```

Comment:
Why are you filtering classes using bbox?

Reply:
Because the displayed classes have to be responsive to the filters performed.
We have two categories of filters (tabular and spatial).
That's why you have to keep.